master/docs/developer/cls-forcesched.rst

```diff
@@ -69,7 +73,7 @@ This section contains information to help users or developers who are interested
 
            A list of types that the parameter conforms to.
            These are used by the jinja template to create appropriate html form widget.
-           The available values are visible in :src:`master/buildbot/status/web/template/forms.html` in the ``force_build_one_scheduler`` macro.
+           The available values are visible in ``master/buildbot/status/web/template/forms.html`` in the ``force_build_one_scheduler`` macro.
```

Comment:
Yes, `forms.html` has been removed.
I opened ticket for tracking update of `ForceScheduler` docs: http://trac.buildbot.net/ticket/3477